pytorch_lightning/trainer/supporters.py

```diff
@@ -279,14 +294,14 @@ def _calc_num_data(datasets: Union[Sequence, Mapping], mode: str) -> Union[int,
             length: the length of `CombinedDataset`
 
         """
-        if mode not in ['min_size', 'max_size_cycle']:
+        if mode not in CombinedDataset.COMPUTE_FUNCS.keys():
             raise ValueError(f"Invalid Mode: {mode}")
 
```

Comment:
this is already checked in `__init__`?

Reply:
yes, but here it is in the static method...